test-cases/longevity/longevity-1TB-7days-authorization-and-tls-ssl.yaml

```diff
@@ -1,4 +1,4 @@
-test_duration: 10900
+test_duration: 1500
```

Comment:
Also need to revert back. test_duration is set in minutes

Reply:
PR  is not in status "ready for review" because of @roydahan wanted me to run 1TB longevity. So longevity yaml is not part of PR and will be reverted after I'll finish the test. So, this comment is not relevant